src/components/collection_vote_box/CollectionSubmitVoteBox.jsx

```diff
@@ -117,8 +117,13 @@ export default function CollectionSubmitVoteBox(props) {
         setVote(newVote);
       }
       Notifications.showSuccess({text: 'Successfully updated vote'});
-    } catch (_error) {
-      Notifications.showError({text: 'Error: Failed to update vote'});
+    } catch (error) {
+      if (error && error.status === 409) {
+        Notifications.showError({text: error.response.data.message + ' Chair vote not submitted, updating chair votes.'});
```

Comment:
minor wording nit: `updating chair votes` could be a bit confusing/ambiguous to a user since it seemingly contradicts the text preceding it

I really like the wording you chose in the other error message below for rationales because it lets the user know that it's just the content on the screen that's being updated/refreshed. Maybe something like `Chair vote not submitted, updating vote display.`?

Reply:
Thanks for the feedback.

The AC had the specific wording for what to display, and I'll always use that when provided.  The other case below wasn't provided so I played with the wording and am glad you noticed the change.  Since we've three of us on board with the change, I'll make it!

Reply:
Also, can this error occur in non-chair votes?  If so, a refinement might help:

```suggestion
        const voteKind = isChair ? 'Chair vote' : 'Vote'
        Notifications.showError({text: `$error.response.data.message} ${voteKind} not submitted, updating vote display.`});
```